app/assets/javascripts/appsignal.coffee

```diff
@@ -0,0 +1,23 @@
+class @Appsignal
+  sendError: (error) ->
+    data = {
+      message:   error.message
+      name:      error.name
+      backtrace: error.stack.split("\n")
+      path:      window.location.pathname
+      environment: {
+        agent:         navigator.userAgent
+        platform:      navigator.platform
+        vendor:        navigator.vendor
+        screen_width:  screen.width
+        screen_height: screen.height
+      }
+    }
+
+    xhr = new XMLHttpRequest()
```

Comment:
Heb deze code uit de [http://docs.appsignal.com/tweaks-in-your-code/frontend-error-catching.html](AppSignal documentatie) gepakt, leek me voordeliger dan om zelf nieuwe code te schrijven & testen.

Zal het in ieder geval nog wel even naar vendor verplaatsen met een comment